cli/formatter/format.go

```diff
@@ -0,0 +1,788 @@
+package formatter
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/apex/log"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
+	"gopkg.in/yaml.v2"
+)
+
+// MakeOutput returns formatted output depending on specified output format and formatting options.
+func MakeOutput(data string, outputFormat Format, opts *Opts) string {
```

Comment:
Does `MakeOutput` works in fact with a different input data format? We have `console_eval_func_body_tbls_fmt.lua` and `console_eval_func_body_fmt.lua`. Or the input data format the same?

Reply:
We get yaml from the server all the time (server works only with `\set output yaml` all the time).
Then, depending on the `outputFormat Format`, a specific render is made by MakeOutput. 

Formats can use their own logic in files like `console_eval_func_body_tbls_fmt.lua` or `console_eval_func_body`, all this points is agreed with the product team.
Through such files, among other things, a compatibility layer will be implemented later for named headers for tuples (cdata) when working with tarantool ver < than 3.

Reply:
The `MakeOutput` function always accepts a yaml formatted string from the server. But the logic of forming this yaml string depends on lua files above

Reply:
Could I use `console_eval_func_body.lua` result for `TableFormat` and `TTableFormat` and `console_eval_func_body_tbls_fmt.lua` for `DefaultFormat`, `YamlFormat` and `LuaFormat`? Why not?

Reply:
Note: At the moment, it has been discussed separately along with the unit test cases to MakeOutput